test/utils/wasm_engine.cpp

```diff
@@ -2,6 +2,8 @@
 // Copyright 2020 The Fizzy Authors.
 // SPDX-License-Identifier: Apache-2.0
 
+#include <test/utils/wasm_engine.hpp>
```

Comment:
```suggestion
#include "wasm_engine.hpp"
```

Reply:
Used this style because it is used in other files in this directory.

Reply:
It should be always "" if including the header matching the .cpp file (clang-format will place it as the first one). Unless the file is not in the same dir.